Gopkg.toml

```diff
@@ -32,3 +32,7 @@
 [[constraint]]
   branch = "master"
   name = "golang.org/x/crypto"
+
+[[constraint]]
+  name = "github.com/alexellis/k3sup"
```

Comment:
This is also not right. The code does not need to vendor itself. Can you remove or 3?